### ISSUE TITLE
conformer_ctc small  config file

### ISSUE THREAD

**Issue by BuaaAlban** [NVIDIA/NeMo#3211]
Hi, will you supply the config for training conformer-ctc samll on librispeech?
https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_conformer_ctc_small_ls

**Comment by VahidooX** [comment]
You may download the model, load it and check the configs. It is the same config here https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_ctc_bpe.yaml but with the changes specified in the table. The effective batch size was also 2K like the original Conformer paper.

**Comment by BuaaAlban** [comment]
> You may download the model, load it and check the configs. It is the same config here https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_ctc_bpe.yaml but with the changes specified in the table. The effective batch size was also 2K like the original Conformer paper.

Thanks a lot for your reply, I noticed that the default batch size in the config is 16 and accumulate_grad_batches=1.
And the effective batch size is 2k, does this means that I should augment it as much as possible (if GPU allows) to make batch_size * accumulate_grad_batches =2k (which is 16 now) ?
Will this have great effect on the final results?

**Comment by titu1994** [comment]
You should lower the learning rate slightly to suit the smaller global batch size. I would suggest about 1/2 - 1/5th the original training value.

**Comment by BuaaAlban** [comment]
I also noticed that the default token type is "wpe", shouldn't it be 'bpe'  ?  
I just want to training this model from scratch
https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_conformer_ctc_small_ls

**Comment by titu1994** [comment]
We support bpe and wpe, for Librispeech WPE is good but for any other dataset we suggest using Sentencepiece (BPE) or Unigram.

**Comment by BuaaAlban** [comment]
The released conformerctc-small model  used BPE (SentencePieceTokenizer), which one is better for Librispeech?

**Comment by titu1994** [comment]
For just Librispeech, wpe is fine. For anything else, Sentencepiece

**Comment by BuaaAlban** [comment]
This is my config  [conformer_ctc_bpe_small_bs2000.txt](https://github.com/NVIDIA/NeMo/files/7657386/conformer_ctc_bpe_small_bs2000.txt) to reproduce the [released one](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_conformer_ctc_small_ls)
I have changed the time masks from 10 to 5 and batch size=16 and accumulate_grad_batches=63 to make the effective batchisze to 2k. Using 4 gpus with 1 node. 

It onlys reach to **WER=5.7%** @epoch192  which is **far from** the released one with WER=3.3%, would you please help to check the config and give some suggestions to reproduce the released accuracy? 

BTW,  the one with bs32 and accu_batches =4 with effective bs 128 seems to converge faster than the one with bs 2k, it achieves WER=0.045@epoch139 and WER=0.426@epoch267.

**Comment by VahidooX** [comment]
https://github.com/NVIDIA/NeMo/issues/3288
